### PR TITLE
Remove force unwrapping in ActionSheetView.swift

### PR DESCRIPTION
**Pull request by cruisediary**

### CODE REVIEW

File: Source/Views/ActionSheetView.swift [hunk 1]
```diff
@@ -109,18 +109,16 @@ final class ActionSheetView: UIView, AlertControllerViewRepresentable {
...
-    class func image(with color: UIColor) -> UIImage {
+    class func image(with color: UIColor) -> UIImage? {
+        defer { UIGraphicsEndImageContext() }
+
         let rect = CGRect(x: 0, y: 0, width: 1, height: 1)
 
         UIGraphicsBeginImageContextWithOptions(rect.size, false, 0)
 
-        let context = UIGraphicsGetCurrentContext()!
+        guard let context = UIGraphicsGetCurrentContext() else { return nil }
```
Comment (sberrevoets) [inline Source/Views/ActionSheetView.swift]: Can you put the code inside {} on a new line?

```swift
guard let context = UIGraphicsGetCurrentContext() else {
	return nil
}
```

**Comment by sberrevoets** [comment]
Can you squash your commits please?

**Comment by cruisediary** [comment]
Ok I'll squash commits at home after working :)

**Comment by sberrevoets** [comment]
Great, thank you!

**Pull request merged**